docs/user/advanced.rst

```diff
@@ -717,10 +717,9 @@ If ``chardet`` is installed, ``requests`` uses it, however for python3
 library is an LGPL-licenced dependency and some users of requests
 cannot depend on mandatory LGPL-licensed dependencies.
 
-When you install ``request`` without specifying ``[use_chardet_on_py3]]`` extra,
+When you install ``requests`` without specifying ``[use_chardet_on_py3]]`` extra,
```

Comment:
Not directly related to your change but while we’re here can we remove the erroneous `]` at the end of the extra?

Reply:
Absolutely! Sorry I missed that.